src/NuGet.Core/NuGet.Versioning/PublicAPI.Shipped.txt

```diff
@@ -149,6 +158,7 @@ static NuGet.Versioning.SemanticVersion.operator ==(NuGet.Versioning.SemanticVer
 static NuGet.Versioning.SemanticVersion.operator >(NuGet.Versioning.SemanticVersion! version1, NuGet.Versioning.SemanticVersion! version2) -> bool
 static NuGet.Versioning.SemanticVersion.operator >=(NuGet.Versioning.SemanticVersion! version1, NuGet.Versioning.SemanticVersion! version2) -> bool
 static NuGet.Versioning.VersionComparer.Compare(NuGet.Versioning.SemanticVersion? version1, NuGet.Versioning.SemanticVersion? version2, NuGet.Versioning.VersionComparison versionComparison) -> int
+static NuGet.Versioning.VersionComparer.Get(NuGet.Versioning.VersionComparison versionComparison) -> NuGet.Versioning.IVersionComparer!
```

Comment:
this isn't in the release-6.7-x branch PR, so it's not a public API that has shipped yet, so it wouldn't be a breaking change in case we want to change it before it ships.

* https://github.com/NuGet/NuGet.Client/pull/5353